tcms/testcases/templates/testcases/mutable.html

```diff
@@ -100,6 +100,18 @@
                 </div>
             </div>
 
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1">{% trans "Setup duration" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <!-- https://github.com/koss-lebedev/bootstrap-duration-picker -->
+                </div>
+
+                <label class="col-md-1 col-lg-1">{% trans "Testing duration" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <!-- https://github.com/koss-lebedev/bootstrap-duration-picker -->
+                </div>
+            </div>
```

Comment:
Remove these changes for now since they need more work. You can back them up locally to save yourself some typing for your next PR.